packages/create-whimsy/README.md

```diff
@@ -0,0 +1,9 @@
+# Create Whimsy
+
+A package used to easily create your whimsy writer project.
+
+Run the following command:
+
+```bash
+npm create whimsy
+```
```

Comment:
Try get better create-whimsy readme.

Maybe also add a --help arg for the create-whimsy bin